bin/fetch_configlet.dart

```diff
@@ -0,0 +1,134 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:archive/archive.dart';
+import 'package:dart2_constant/convert.dart';
+
+final HttpClient httpClient = new HttpClient();
+
+final String osString = Platform.operatingSystem;
+final bool isWindows = osString == 'windows';
+
+void main() async {
+  final String githubApi = 'https://api.github.com/repos/exercism/configlet/releases?per_page=1';
+  String archString = '64bit';
+
+  print('Trying to detect OS and Architecture');
+  try {
+    final Process getArch = await Process.start('uname', ['-m']);
```

Comment:
there exists a Windows equivalent command, that would remove the dependency on bash